LOAN/LoansTemporal/LoanApplicationsTemporal.rdf

```diff
@@ -79,34 +86,76 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-tem-app;LoanApplicant"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-arr-com;Applicant"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-app;appraisedBy"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isProvidedBy"/>
 				<owl:someValuesFrom rdf:resource="&fibo-loan-tem-app;LoanAppraiser"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-app;mayBecome.1"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-loan;LoanDrawdown"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
 		<owl:disjointWith rdf:resource="&fibo-loan-tem-ph;DisbursedLoan"/>
 		<owl:disjointWith rdf:resource="&fibo-loan-tem-ph;RepaidLoan"/>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-tem-app;LoanApplicant">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-com;Applicant"/>
+	<owl:Class rdf:about="&fibo-loan-tem-app;BorrowerInformation">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-doc;Record"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:onClass rdf:resource="&fibo-loan-ln-ln;PaymentHistory"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:onClass rdf:resource="&fibo-loan-tem-app;IncomeVerificationReport"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-app;mayBecome"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-pty;LoanBorrower"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:onClass rdf:resource="&fibo-loan-tem-app;IndividualPersonCreditRating"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">loan applicant</rdfs:label>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-doc;isAbout"/>
+				<owl:onClass rdf:resource="&fibo-loan-tem-ph;DisbursedLoan"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isProvidedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-pty;LoanServicer"/>
```

Comment:
this relates to the Applicant - so what is provided here?

Reply:
Again - this is one of those that I started working my way through - haven't finished doing that work.  We can review in another pass.